### PR TITLE
chore: dockerization

### PR DESCRIPTION
**Pull request by amordzon**

### CODE REVIEW

File: client/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM node:20-alpine as builder
+WORKDIR /app
+COPY ./package.json package-lock.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+FROM nginx:stable-alpine
```
Comment (jtowarnicka) [inline client/Dockerfile:8]: shouldn't we use specified version on nginx instead of latest (idk what version it is)?

File: server/gradlew
Comment (jtowarnicka) [inline server/gradlew:1]: i think it's a leftover

File: client/default.conf
Comment (jtowarnicka) [inline client/default.conf:1]: check out overwriting /etc/nginx/templates/default.conf.template

File: server/gradlew
Comment (amordzon) [reply server/gradlew:1]: YOU TRICKED ME

Comment (jtowarnicka) [reply server/gradlew:1]: I THOUGHT IT WAS EMPTY

File: client/Dockerfile (hunk 1 above)
Comment (amordzon) [reply client/Dockerfile:8]: nope hehe

**Review by piotrd22 — APPROVED** [review]
lgtm

**Pull request merged**